src/content/enterprise/eks.mdx

```diff
@@ -145,29 +128,12 @@ Want to use Okteto with a bigger team? [Let's talk](https://okteto.com/#talktous
 
 ## Installing Okteto
 
-### Creating the Okteto namespace
-
-Create a dedicated namespace for your Okteto instance:
-
-```console
-$ kubectl apply -f https://charts.okteto.com/namespace.yaml
-```
-
-*The `namespace.yaml` manifest assumes you'll be installing the application into the `okteto` namespace. If you are not doing this, you'll need to update the file before creating the namespace.*
-
-Jump to the next step if you are [bringing your own certificates](/docs/enterprise/administration/certificates/).
-
-If you are already running Cert Manager in your cluster, add this setting to your Okteto helm configuration:
-
-```
-cert-manager:
-  enabled: false
-```
+### Create the Okteto Namespace
```

Comment:
I don't think we need an extra step here, just include it in the helm install part.

Reply:
A few secrets must be created after creating the namespace and before installing okteto. We will improve this part on a different iteration